site/content/en/docs/Getting Started/create-fleet.md

```diff
@@ -76,56 +76,6 @@ Let's wait for the two `GameServers` to become ready.
 watch kubectl describe fleet simple-udp
 ```
 
-{{% feature expiryVersion="0.12.0" %}}
-```
-Name:         simple-udp
-Namespace:    default
-Labels:       <none>
-Annotations:  kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"stable.agones.dev/v1alpha1","kind":"Fleet","metadata":{"annotations":{},"name":"simple-udp","namespace":"default"},"spec":{"replicas":2,...
-API Version:  stable.agones.dev/v1alpha1
```

Comment:
Why removing the examples? Shouldn't them be replaced with v1 version of the same example?

Reply:
yep - that's the next line down after this. This is removing and cleaning up the pre-v1 example

Reply:
The v1 versions are live. See https://agones.dev/site/docs/getting-started/create-fleet/